internal/cmp/pool_datasource.go

```diff
@@ -0,0 +1,49 @@
+// (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+package cmp
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pkg/client"
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/utils"
+	"github.com/tshihad/tftags"
+)
+
+type poolds struct {
+	lbClient *client.LoadBalancerAPIService
+}
```

Comment:
please add lb_ as a prefix for the filename and wherever necessary. People will get confused with network pools

Reply:
done